.gitignore

```diff
@@ -17,5 +17,4 @@ distribute-*.tar.gz
 up.bat
 anthrocon_plugin/
 add_table_extras.sql
-alembic*
```

Comment:
Yeah, sorry. I was definitely disappointed when I realized that GItHub didn't edit out the stuff that was in a previous PR, even after that PR was merged into ac-staging. :\ I thought if I based one feature branch off the previous one, that'd work. Buh.